scripts/create-package.sh

```diff
@@ -74,11 +74,16 @@ install_doc()
             ;;
     esac
     # Fill template variables in README.template
-    if [ -n "$git_commit" ]; then
-        sed -i -e "s|%GIT_COMMIT%|$git_commit|" "$readme_tmpl"
+    if [[ "$git_branch" == "refs/tags/"* ]] && [[ "$git_branch" != *"-"* ]]; then
+        export version_tag=`echo $git_branch | awk '{print substr($0,11);exit}'`
+        export package_information="release $version_tag"
+    elif [ -n "$git_branch" ] && [ -n "$git_commit" ]; then
+        export package_information="a development branch named $git_branch with commit $git_commit"
+    else
+        export package_information="a development branch"
```

Comment:
The `export` keyword will place the `package_information` variable in the calling shell's environment (is this an intended side-effect of the script?).

If it's not intended, then we can drop the `export` and use local variables.

(Sorry for the delay on this comment @Wengier , I had it queued up yesterday but didn't submit it).

Reply:
Thanks for pointing this out. I removed the export keyword, which hopefully makes the variables local.